src/api/app/controllers/webui/user_controller.rb

```diff
@@ -57,11 +57,8 @@ def show
     @owned = @displayed_user.owned_packages
 
     if User.current == @displayed_user
-      @reviews = @displayed_user.involved_reviews
+      @reviews = @displayed_user.involved_reviews.count
```

Comment:
Why not 
```
@reviews = @displayed_user.involved_reviews.exists?
```
and then you can do in the view
```
<% if @reviews %>
```

instead of 
```
<% if @reviews && @reviews > 0 %>
```
```